libraries/AP_HAL_ChibiOS/hwdef/skyviper-v2450/hwdef.dat

```diff
@@ -73,8 +73,7 @@ define GPS_MOVING_BASELINE 0
 define GRIPPER_ENABLED 0
 define HAL_ADSB_SAGETECH_ENABLED 0
 define HAL_ADSB_UAVIONIX_MAVLINK_ENABLED 0
-define AP_AIS_ENABLED 0
-define AP_BATTMON_FUEL_ENABLE 0
+define HAL_AIS_ENABLED 0
```

Comment:
needs to be AP_AIS_ENABLED

Reply:
Not sure when or why I might have done that. Fixed